src/slices/GiveThunk.ts

```diff
@@ -142,8 +140,13 @@ export const getTestTokens = createAsyncThunk(
       return;
     }
 
+    if (networkID !== 4) {
+      dispatch(error("Feature only available on Rinkeby"));
+      return;
+    }
+
     const signer = provider.getSigner();
-    const mockSohmContract = new ethers.Contract(addresses[networkID].MOCK_SOHM as string, MockSohm, signer);
+    const mockSohmContract = new ethers.Contract(addresses[4].MOCK_SOHM as string, MockSohm, signer);
```

Comment:
This doesn't seem right to hardcode it. You already have a guard earlier on, so leaving it as networkID seems fine.